lib/plugins/aws/invokeLocal/fixture/asyncHandlerWithError.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-module.exports.withError = () => Promise.reject(new Error('failed'));
```

Comment:
Did you verify that these are correctly used if they no longer return a promise? The name of the fixture `asyncHandlerWithError` suggests that it's expected that these handlers are asynchronous

Reply:
I went through and removed await from all the places that made these calls in this update.

Reply:
I did, these two files are only used within the context of unit testing. Since one of the exported functions still returns a promise: `asyncHandlerWithSuccess.withMessageAndDelayByCallback`, all paths of `invokeLocal()` are tested.

That said, if you'd like me to revert it, I'd be more than happy to.

Reply:
Thanks for confirming :+1: